core/platforms/packaging/bareos.spec

```diff
@@ -708,29 +706,27 @@ Requires: php-xmlreader
 Requires: php-xmlwriter
 Requires: php-zip
 
-%if 0%{?suse_version} || 0%{?sle_version}
-BuildRequires: apache2
-# /usr/sbin/apxs2
-BuildRequires: apache2-devel
-BuildRequires: mod_php_any
-%define _apache_conf_dir /etc/apache2/conf.d/
-%define www_daemon_user  wwwrun
-%define www_daemon_group www
-Requires: apache
-Recommends: mod_php_any
-%else
-BuildRequires: httpd
-# apxs2
-BuildRequires: httpd-devel
+%if !0%{?suse_version}
+Requires: php-libxml
```

Comment:
Beware that one has disappear with new php8 packaging.

Reply:
Thanks for the hint.